crates/sui-core/src/authority/execution_time_estimator.rs

```diff
@@ -34,7 +35,7 @@ use tracing::{debug, info, trace, warn};
 
 // TODO: Move this into ExecutionTimeObserverConfig, if we switch to a moving average
 // implmentation without the window size in the type.
-const LOCAL_OBSERVATION_WINDOW_SIZE: usize = 20;
+const SMA_LOCAL_OBSERVATION_WINDOW_SIZE: usize = 20;
```

Comment:
oh right that's why this was a const, because the SMA library makes it a generic param